website/content/docs/configuration/listener/index.mdx

```diff
@@ -9,9 +9,181 @@ description: |-
 # `listener` stanza
 
 The `listener` stanza configures the addresses and ports on which Vault will
-respond to requests. At this time, there are two listeners: 
+respond to requests. At this time, there are two listeners:
 - [TCP][tcp]
 - [Unix Domain Socket][unix]
 
 [tcp]: /vault/docs/configuration/listener/tcp
 [unix]: /vault/docs/configuration/listener/unix
+
+## Unauthenticated endpoints - sensitive data redaction
+
+There are four different types of information deemed sensitive that can be returned
+by unauthenticated API endpoints:
+
+1. Version number
+2. Build date
+3. Cluster name
+4. IP address
+
+Vault offers the ability to configure each `listener` stanza such that when appropriate,
+these values will be redacted from responses.
+
+The following API endpoints support redaction based on `listener` stanza configuration:
+
+* [`/sys/health`](/vault/api-docs/system/health)
+* [`/sys/leader`](/vault/api-docs/system/leader)
+* [`/sys/seal-status`](/vault/api-docs/system/seal-status)
+
+When a value is redacted by Vault, it will be replaced with an empty string (`""`).
+
+~> Note: In certain situations, due to an empty string value, the related key may
+no longer be present in the response object returned from the API, as that field is
+omitted when the value is empty (`""`).
+
+~> Note: The Vault API is also consumed by the Vault CLI and UI, therefore enabling redaction
+settings will also affect them both.
+
+## Redaction examples
+
+Please see [redaction settings](/vault/docs/configuration/listener#redaction-settings)
+(below) for details on each redaction setting.
+
+Example configuration for the [`tcp`](/vault/docs/configuration/listener/tcp) listener,
+enabling [`redact_addresses`](/vault/docs/configuration/listener#redact_addresses),
+[`redact_cluster_name`](/vault/docs/configuration/listener#redact_cluster_name) and
+[`redact_version`](/vault/docs/configuration/listener#redact_version).
```

Comment:
In general, we should introduce examples with plain language that explains the outcome. In this case, we really don't need to explicitly list the relevant parameters since (1) the name makes it pretty obvious and (2) if we switch things around so the settings are first in the doc, in theory, the reader will already have an idea of what they're looking at and which parameters matter to the example :)  

```suggestion
Configure the [`tcp`](/vault/docs/configuration/listener/tcp) listener to
redact IP address, cluster name, and version information:
```

Style correction: write in the active voice, use declarative language to introduce instruction or examples.